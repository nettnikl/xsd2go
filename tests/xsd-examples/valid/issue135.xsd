<?xml version="1.0" encoding="UTF-8"?>
<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
            elementFormDefault="qualified" attributeFormDefault="qualified"
            version="1.0">
    <xsd:complexType name="AAA">
        <xsd:sequence>
            <xsd:element name="BBB" minOccurs="1" maxOccurs="1">
                <xsd:simpleType>
                    <xsd:restriction base="xsd:string"/>
                </xsd:simpleType>
            </xsd:element>
            <xsd:sequence>
                <xsd:element name="CCC" minOccurs="1" maxOccurs="unbounded">
                    <xsd:simpleType>
                        <xsd:restriction base="xsd:string"/>
                    </xsd:simpleType>
                </xsd:element>
            </xsd:sequence>
        </xsd:sequence>
    </xsd:complexType>
</xsd:schema>
