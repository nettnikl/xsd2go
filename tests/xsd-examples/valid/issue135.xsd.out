// Code generated by https://github.com/gocomply/xsd2go; DO NOT EDIT.
// Models for
package issue135

import (
	"encoding/xml"
)

// XSD ComplexType declarations

type Aaa struct {
	XMLName xml.Name

	Bbb string `xml:"BBB"`

	Ccc []string `xml:"CCC"`
}

// XSD SimpleType declarations
